### ISSUE TITLE
Invalid RSS feed URL breaks RSS Bot

### ISSUE THREAD

**Issue by r4dh4l** [vector-im/element-web#16094]
Hi,

I've added some RSS feed links to the Matrix/Element RSS Bot integration which are not valid anymore. Choosing "Unsubscribe" to remove these URLs Element reports

> HTTP 500: Failed to register service: Failed to read URL https://...: Failed to detect feed type

Furthermore it is not possible to add other RSS feed links in this situation.

**Comment by jryans** [comment]
Yes, it's quite unfortunate. I believe https://github.com/matrix-org/go-neb/issues/315 is the canonical issue.
